src/main/java/com/parasoft/findings/sonar/ParasoftConfiguration.java

```diff
@@ -28,6 +30,17 @@ public static List<PropertyDefinition> getProperties()
                 .multiValues(true)
                 .build());
         }
+        // Support Code Coverage
+        items.add(PropertyDefinition.builder(ParasoftConstants.COVERAGE_REPORT_DIR_KEY)
+                .name("Parasoft coverage report path")
+                .description("Path to parasoft coverage report.")
```

Comment:
Change the description to "Path (absolute or relative) to parasoft coverage xml report files."